app/controllers/application_controller.rb

```diff
@@ -46,6 +47,11 @@ def create_user_event(event_type, user = current_user)
 
   private
 
+  def disable_caching
+    response.headers['Cache-Control'] = 'no-store'
+    response.headers['Pragma'] = 'no-cache'
```

Comment:
According to this article, the settings I mentioned are the most secure, recommended ones due to the way different browsers interpret the headers:
https://richhewlett.com/2015/02/16/preventing-browser-caching-using-http-headers/

> Below are the settings seen on many secure sites as a comparison to above and perhaps as a guide to what we should really be aiming for:

> Cache-Control:max-age=0, no-cache, no-store, must-revalidate 
Expires:Thu, 01 Jan 1970 00:00:00 GMT 
Pragma:no-cache